go/api/hyperloglog_commands.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+// HyperLogLogCommands defines an interface for the "HyperLogLog Commands" group of commands for standalone and cluster
+// clients.
+//
+// [valkey.io]: https://valkey.io/commands/?group=string#hyperloglog
```

Comment:
Please fix the link and align the doc with #2844

Reply:
done